docker-compose.yml

```diff
@@ -3,12 +3,12 @@ services:
   redis:
     image: redis
     ports:
-      - "127.0.0.1:6379:6379"
```

Comment:
Please remove changes

Reply:
Is it intentional that the Postgress port was set to "127.0.0.1:5532:5432" instead of "127.0.0.1:5432:5432" (notice the 5 instead of 4 in the first one). Build was failing for me bc of that.